src/DataBox/docs/New-AzDataBoxJob.md

```diff
@@ -68,12 +69,24 @@ AddressType City          CompanyName Country PostalCode StateOrProvince StreetA
 Commercial  San Francisco             US      94107      CA              101 TOWNSEND ST
 ```
 
-You can expand and visualize other object in similar way how details and shipping address expanded
+You can expand and visualize other object in similar way how details and shipping address expanded, Also you can use $DebugPreference = "Continue", with any example/usecase to get exact details of error in below format when command fails.
+```json
+{
+  "Error": {
+    "Code": "StaticValidationGenericCountryCodeHasInvalidLength",
+    "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.",
+    "Details": [
+      null
+    ],
+    "Target": null
+  }
+} 
+```
```

Comment:
The tool to process help document does not like multiple code blocks in one example unfortunalty. @dhawal777  Would you mind revising it as below?
```suggestion
You can expand and visualize other object in similar way how details and shipping address expanded, Also you can use `$DebugPreference = "Continue"`, with any example/usecase to get exact details of error in below format when command fails.

> {
>   "Error": {
>     "Code": "StaticValidationGenericCountryCodeHasInvalidLength",
>     "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.",
>     "Details": [
>       null
>     ],
>     "Target": null
>   }
> } 
```

Reply:
Will appear like this , can we have some way to keep format intact

{ "Error": { "Code": "StaticValidationGenericCountryCodeHasInvalidLength", "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.", "Details": [ null ], "Target": null } }

Reply:
Unless we add it to the example code as comments, like this:

### Example 1: Create a databox import job 
```powershell
PS C:\> $DebugPreference = "Continue"
***********
# You can expand and visualize other object in similar way how details and shipping address expanded, Also you can use `$DebugPreference = "Continue"`, with any example/usecase to get exact details of error in below format when command fails.
# {
#   "Error": {
#     "Code": "StaticValidationGenericCountryCodeHasInvalidLength",
#     "Message": "The attribute country code does not meet length constraints.\r\nEnter a value with 2 characters for country code.",
#     "Details": [
#       null
#     ],
#     "Target": null
#   }
# } 
```

What do you think?

Reply:
Yes LGTM, Thanks